README.md

```diff
@@ -96,9 +96,9 @@ want to limit this to just one.
 The `--project` argument will be matched case insensitively against all PT
 projects accessible to you.
 
-You can also omit this parameter and specify the project name via the
+You can also omit this option and specify the project name via the
```

Comment:
This should be "argument" to be consistent with the previous paragraph.

Reply:
I was torn about this. The heading calls them command line options. In my opinion, the option is `--project "Project Name"` and the argument is `"Project Name"` so I think we should use argument here.